cardano-cli/src/Cardano/CLI/EraBased/Commands/Governance.hs

```diff
@@ -48,8 +47,6 @@ data GovernanceCmds era
       (GovernanceCommitteeCmds era)
   | GovernanceDRepCmds
       (GovernanceDRepCmds era)
-  | GovernanceHashCmds
```

Comment:
I'm not sure I would break this immediately. Better to deprecate it first.

Reply:
No need to go through a deprecation period, as per @CarlosLopezDeLara's comment below :point_down: